scripts/update-dependencies/UpdateFilesTargets.cs

```diff
@@ -0,0 +1,233 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Text.RegularExpressions;
+using Microsoft.DotNet.Cli.Build.Framework;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NuGet.Versioning;
+
+namespace Microsoft.DotNet.Scripts
+{
+    public static class UpdateFilesTargets
+    {
+        private static HttpClient s_client = new HttpClient();
+
+        [Target(nameof(GetDependencies), nameof(ReplaceVersions))]
+        public static BuildTargetResult UpdateFiles(BuildTargetContext c) => c.Success();
+
+        /// <summary>
+        /// Gets all the dependency information and puts it in the build properties.
+        /// </summary>
+        [Target]
+        public static BuildTargetResult GetDependencies(BuildTargetContext c)
+        {
+            string coreFxLkgVersion = s_client.GetStringAsync("https://raw.githubusercontent.com/eerhardt/versions/master/corefx/release/1.0.0-rc2/LKG.txt").Result;
```

Comment:
Oops.  Missed this one that needs to be parameterized.  I'll update this in the morning to read this from an Environment Variable.